frontend/src/api/HttpClientInstance.ts

```diff
@@ -1,7 +1,10 @@
 import {HttpClient} from './HttpClient'
 import { HttpClientWithInterceptors } from './httpClientWithInterceptors'
 import {OperateUserData} from './operatePreferenceData'
+import {CustomizationRequest} from './customizationRequest'
 
 export const httpClient = new HttpClient()
 export const httpClientWithInterceptors = new HttpClientWithInterceptors()
-export const operateUserDataRequest = new OperateUserData(httpClientWithInterceptors)
\ No newline at end of file
+export const operateUserDataRequest = new OperateUserData(httpClientWithInterceptors)
+
```

Comment:
I think `OperateUserData` and `CustomizationRequest` have same scope and should be same class